### PR TITLE
Announce pulp_container in Pulp 3.

### PR DESCRIPTION
**Pull request by ipanova**

### CODE REVIEW

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Pulp docker plugin will be renamed to Pulp container plugin in Pulp3
```
Comment (goosemania) [inline _posts/2019-11-12-docker-rename-to-container.md]: maybe 
s/docker/Docker/
s/container/Container/
what do you think?

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+author: Ina Panova
+tags:
+  - docker
+  - container
+---
+
+With Pulp 3 GA pulp_container plugin will come along.
```
Comment (goosemania) [inline _posts/2019-11-12-docker-rename-to-container.md]: s/pulp_container/the pulp_container/ ? Ask natives though :)

**Review by goosemania — COMMENTED** [review]
How about mentioning new git repos?
What about versioning? Maybe it's helpful to say what would be the release of pulp_container? Is it 4.0? Is it 0.1?  If it's 0.1, does it imply that it's quite new and unstable? Maybe re-emphasize that the code is stable enough even if the version number is low. (I'm speculating here, since I have no idea what version you plan to use).

Maybe encourage to contribute if anyone is eager to add OCI support themselves?

File: _posts/2019-11-12-docker-rename-to-container.md (hunk 2 above)
Comment (goosemania) [reply _posts/2019-11-12-docker-rename-to-container.md]: I'd ask to check articles everywhere :)

Comment (daviddavis) [reply _posts/2019-11-12-docker-rename-to-container.md]: `the pulp_container plugin` is more correct but `pulp_container plugin` also works. If you refer to pulp_container alone, don't use `the`.

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp Docker plugin will be renamed to Pulp Container plugin in Pulp3
+author: Ina Panova
+tags:
+  - docker
+  - container
+---
+
+With Pulp 3 GA pulp_container 1.0 will come along.
```
Comment (daviddavis) [inline _posts/2019-11-12-docker-rename-to-container.md]: Version 1.0 of the pulp_container plugin will be released to coincide with Pulp 3 GA.

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Why renaming pulp_docker to pulp_container in Pulp3?
```
Comment (daviddavis) [inline _posts/2019-11-12-docker-rename-to-container.md]: Why rename...

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+* pulp_docker plugin implies management of docker format images only. This plugin will stay in Pulp 2.
```
Comment (daviddavis) [inline _posts/2019-11-12-docker-rename-to-container.md]: s/will stay in/will remain/

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* pulp_container won't be tied to docker format images, this will enable us to add support for OCI format in the future.
```
Comment (daviddavis) [inline _posts/2019-11-12-docker-rename-to-container.md]: `the OCI format` or `OCI format images`.

File: _posts/2019-11-12-docker-rename-to-container.md (hunk 5 above)
Comment (daviddavis) [reply _posts/2019-11-12-docker-rename-to-container.md]: I think Docker should be capitalized everywhere. At the very least, capitalization should be consistent throughout.

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+## What does this mean for me?
+
+Pulp 2.y users who manage Docker content should be able to migrate their production
+systems to Pulp 3.0 with the Pulp Container plugin installed.
+Pulp 3 Container plugin has full feature parity with Pulp 2 Docker plugin.
+
+Pulp 3 users will need to uninstall pulp_docker plugin and install pulp_container one.
+Don't be alaramed to see such a low versioning number, this plugin release is stable.
```
Comment (daviddavis) [inline _posts/2019-11-12-docker-rename-to-container.md]: Don't be alarmed by the low version number, this...

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+* pulp_docker plugin implies management of docker format images only. This plugin will stay in Pulp 2.
+* pulp_container won't be tied to docker format images, this will enable us to add support for OCI format in the future.
+
+## What does this mean for me?
+
+Pulp 2.y users who manage Docker content should be able to migrate their production
+systems to Pulp 3.0 with the Pulp Container plugin installed.
+Pulp 3 Container plugin has full feature parity with Pulp 2 Docker plugin.
+
+Pulp 3 users will need to uninstall pulp_docker plugin and install pulp_container one.
```
Comment (daviddavis) [inline _posts/2019-11-12-docker-rename-to-container.md]: Pulp 3 users will need to uninstall the pulp_docker plugin and install the pulp_container one.

File: _posts/2019-11-12-docker-rename-to-container.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+tags:
+  - docker
+  - container
+---
+
+Version 1.0 of the pulp_container plugin will be released to coincide with Pulp 3 GA..
+
+## Why rename pulp_docker to pulp_container in Pulp3?
+
+* pulp_docker plugin implies management of Docker format images only. This plugin will remain in Pulp 2.
```
Comment (RCMariko) [inline _posts/2019-11-12-docker-rename-to-container.md]: "This plugin" --> "This plugin naming convention"

**Review by RCMariko — APPROVED** [review]

**Pull request merged**